.github/workflows/dbld-images.yml

```diff
@@ -55,15 +50,15 @@ jobs:
           gh_export UPLOAD_IMAGES_INTERNAL
 
       - name: Log in to the Container registry
-        if: env.UPLOAD_IMAGES_INTERNAL == 'true'
+        if: always() && env.UPLOAD_IMAGES_INTERNAL == 'true'
         uses: docker/login-action@v1
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Push the images
-        if: env.UPLOAD_IMAGES_INTERNAL == 'true'
+        if: always() && env.UPLOAD_IMAGES_INTERNAL == 'true'
         run: |
           IMAGES=$(docker image ls --filter "reference=${{ env.CONTAINER_REGISTRY }}/dbld-*:latest" --format '{{.Repository}}:{{.Tag}}')
```

Comment:
couldn't we push this functionality into dbld/rules ? I mean we already have a ./dbld/rules pull-image-<whatever>, we could have a push-image or even a push-images target so the github action would become thinner and we would gain the ability to push images easily even when we are working interactively in dbld shell.

Reply:
That sounds like a good idea. I'll add it soon.